spec/automated_review/column_documentation_spec.rb

```diff
@@ -2,11 +2,7 @@
   # Documented tables
   %w[conversion_hosts miq_tasks].each do |table_name|
     it "#{table_name} columns are documented" do
-      klass = Class.new(ActiveRecord::Base) do
-        self.table_name = table_name
-      end
-
-      klass.columns.each do |column|
+      new_migration_stub(table_name).columns.each do |column|
```

Comment:
Might need to find a new place for `.new_migration_stub`, because it isn't available for these specs.

Possible that we could move it out of where I put it and place it somewhere more generic that all spec files have access to.

Reply:
I'll just undo this one for now for this PR.